index.html

```diff
@@ -232,7 +232,7 @@ <h2>
                     <ul>
                         <li>
                             <span class="skills">
-                                Languages <span><img src="./images/Disabled.png" alt="languages"></span>
+                                Languages <span><img src="./images/Disabled.png" alt="Languagess"></span>
```

Comment:
- [ ] You have done great having the `languages, frameworks and skills` titles 👏 . However, for [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) purposes I would suggest using [HTML headings](https://www.w3schools.com/html/html_headings.asp)(I.e `h2`) or paragraph. This increases the accessibility of the website to screen readers. Read through this [Stackoverflow conversation](https://stackoverflow.com/questions/6555014/is-it-semantically-incorrect-to-place-h2-tags-between-li-tags) to have a clear understanding ☺️ .